src/components/theme/Widgets/TokenWidget.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import cx from 'classnames';
+
+const TokenWidget = ({ value, children, className }) =>
+  value ? (
+    <span className={cx(className, 'token', 'widget')}>
+      {value.map((tag) => (
+        <a className="ui label" href={`/search?Subject=${tag}`} key={tag}>
```

Comment:
here too.. use UniversalLink where you do links

Reply:
done:

https://github.com/plone/volto/commit/b83586d074fb501d1773a2271b2a0570dd8446c3